### PR TITLE
Update FTIMULoggerDevice and rename in YarpRobotLoggerDevice

### PR DESCRIPTION
**Pull request by isorrentino**
The aim is to be able to log all the robot sensors launching this single device. The XML files give the possibility to enable or disable sensors data to log.

### CODE REVIEW

**Comment by GiulioRomualdi** [comment]
The `CHANGELOG` ;)

**Comment by isorrentino** [comment]
> The `CHANGELOG` ;)

Every time :(

**Comment by isorrentino** [comment]
Friendly ping @prashanthr05 @GiulioRomualdi :)

File: devices/YarpRobotLoggerDevice/app/robots/iCubGenova02/interface/all_joints_mc.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+This software may be modified and distributed under the terms of the
+GNU Lesser General Public License v2.1 or any later version. -->
+<?xml version="1.0" encoding="UTF-8" ?>
+<device xmlns:xi="http://www.w3.org/2001/XInclude" name="all_joints_mc" type="remotecontrolboardremapper">
+    <!--param name="remoteControlBoards">("/icub/head", "/icub/torso", "/icub/left_arm", "/icub/right_arm", "/icub/left_leg", "/icub/right_leg")</param>
+    <param name="axesNames">("neck_pitch", "neck_roll", "neck_yaw", "torso_pitch", "torso_roll", "torso_yaw", "l_shoulder_pitch", "l_shoulder_roll", "l_shoulder_yaw", "l_elbow", "r_shoulder_pitch", "r_shoulder_roll", "r_shoulder_yaw", "r_elbow", "l_hip_pitch", "l_hip_roll", "l_hip_yaw", "l_knee", "l_ankle_pitch", "l_ankle_roll", "r_hip_pitch", "r_hip_roll", "r_hip_yaw", "r_knee", "r_ankle_pitch", "r_ankle_roll")</param-->
+    <param name="remoteControlBoards">("/icub/right_leg")</param>
+    <param name="axesNames">("r_hip_pitch", "r_hip_roll", "r_hip_yaw", "r_knee", "r_ankle_pitch", "r_ankle_roll")</param>
+    <param name="localPortPrefix">/ft_imu_logger/joints</param>
```
Comment (prashanthr05) [inline devices/YarpRobotLoggerDevice/app/robots/iCubGenova02/interface/all_joints_mc.xml:10]: This file can be renamed as `right_leg_mc.xml` and accordingly the device name.

**Review by prashanthr05 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**